R/aaa-auto.R

```diff
@@ -739,8 +739,9 @@ distances_floyd_warshall_impl <- function(graph, from=V(graph), to=V(graph), wei
   res
 }
 
-voronoi_impl <- function(graph, generators, weights=NULL, mode=c("out", "in", "all", "total"), tiebreaker=c("random", "first", "last")) {
+voronoi_impl <- function(graph, generators, ..., weights=NULL, mode=c("out", "in", "all", "total"), tiebreaker=c("random", "first", "last")) {
   # Argument checks
+  check_dots_empty()
```

Comment:
have we documented this pattern somewhere? it looks familiar but I'm not entirely sure I understand.

Reply:
This is auto-generated, not written manually by me. The purpose is to make sure that any parameters following `...` are specified explicitly, with name. 

https://rlang.r-lib.org/reference/check_dots_empty.html

See #1045

CC @krlmlr

Reply:
Yes, the `*` in `functions-R.yaml` .